TerminalDocs/customize-settings/key-bindings.md

```diff
@@ -22,12 +22,24 @@ Key bindings can be structured in the following formats:
 { "command": "commandName", "keys": "modifiers+key" }
 ```
 
+For example, this default setting uses the shortcut key <kbd>alt+f4</kbd> to close the terminal window:
+
+```json
+{ "command": "closeWindow", "keys": "alt+f4" }
+```
+
 ### Commands with arguments
 
 ```json
 { "command": { "action": "commandName", "argument": "value" }, "keys": "modifiers+key" }
 ```
 
+For example, this default setting uses the shortcut key "ctrl+shift+1" to open a new tab in the terminal based on whatever command line is listed first in your dropdown menu (typically this will open the PowerShell profile):
```

Comment:
```suggestion
For example, this default setting uses the shortcut key <kbd>ctrl+shift+1</kbd> to open a new tab in the terminal based on whichever profile is listed first in your dropdown menu (typically this will open the PowerShell profile):
```